.travis.yml

```diff
@@ -1,5 +1,10 @@
+language: java
```

Comment:
why do you need to explicitly declare java language?

Reply:
from what I tested, this makes it possible to use `jdk: openjdk17` to select the Java version